hoodie-hive/src/main/java/com/uber/hoodie/hive/HiveSyncTool.java

```diff
@@ -61,15 +64,38 @@ public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
   }
 
   public void syncHoodieTable() {
-    LOG.info("Trying to sync hoodie table" + cfg.tableName + " with base path " + hoodieHiveClient
+    switch(hoodieHiveClient.getTableType()) {
+      case COPY_ON_WRITE:
+        syncHoodieTable(false);
+        hoodieHiveClient.close();
```

Comment:
I meant why repeat this on every case instead of doing it once outside the switch once.